doc/_admin-guide/190_The_syslog-ng_manual_pages/010_syslog-ng_conf.md

```diff
@@ -251,19 +249,19 @@ listed below. For details, see the
 |amqp()|           Publishes messages using the AMQP (Advanced Message Queuing Protocol).
```

Comment:
i was wondering if it broke anything adding | to the pattern separators to find title occurrences in these table declarations as well